### ISSUE TITLE
init statement and init phase are not documented

### ISSUE THREAD

**Issue by MiiNiPaa** [renpy/renpy#3639]
Init statement used to be documented in Quickstart section, but now it is absent from documentation (only `init python` is documented), only broken links on forums and entry in historic wiki remains.

Likewise there are multiple references to "init time" and "init phase", but no explanation provided (before it used to be shortly explained in init statement documentation, explanation of order of init blocks execution currently present in `init python` documentation)

**Comment by Gouvernathor** [comment]
Could you provide one of such broken links, so that we can try to patch them up correctly ?

**Comment by MiiNiPaa** [comment]
Here is the example post which links to what was `init` documentation:
https://lemmasoft.renai.us/forums/viewtopic.php?p=427270#p427270

Here the link itself:
https://www.renpy.org/doc/html/quickstart.html?highlight=init#init

**Comment by Gouvernathor** [comment]
Due to the way the link is built, I don't see how it could be fixed other than by re-adding an init section in the quickstart page (which is not the way I did it in the PR). I'm not going to simply readd it without @renpytom's request, because if he removed it it was for a good reason.
Maybe adding a very small section, just for that purpose, to say in essence "init: it used to be documented here but now it's over there" ? But it would feel out of place with the rest of the page.
Or maybe adding a bottom "See also" section, including a specific link to the init phase section of the python page (like the one I added in my PR) ? And adding an invisible "init" anchor to that See also section, so that the existing links lead to it. It would not be as perfect as a redirect (which can't be done), but still better than having the existing links to to the totally unrelated top of the Quickstart page.
